### ISSUE TITLE
Templating the start of a project

### ISSUE THREAD

**Issue by pdonorio** [rapydo/issues#26]
Currently we make the creation of a new project based on forking the [`rapydo/core`](https://github.com/rapydo/core) repo.

Usually a much better option is templating. Of course templating in python means [`jinja2`](http://jinja.pocoo.org/) engine. However much more interesting option exists. The most wide used is a project based on jinja, called [`cookiecutter`](https://cookiecutter.readthedocs.io/en/latest/overview.html).

To work with cookiecutter we might [create on the rapydo side a project template ](https://cookiecutter.readthedocs.io/en/latest/first_steps.html) which other user may then use for generate the project. Note: we still might need to create a git repository to give cookiecutter a template, but at least we will finally use some standards.

We can generate the project within Python in `do`, see [src](https://cookiecutter.readthedocs.io/en/latest/readme.html#features):
```python
from cookiecutter.main import cookiecutter

# Create project from the cookiecutter-pypackage/ template
cookiecutter('cookiecutter-pypackage/')

# Create project from the cookiecutter-pypackage.git repo template
cookiecutter('https://github.com/audreyr/cookiecutter-pypackage.git')
```

**Comment by pdonorio** [comment]
Now we can discuss how to create a better way of updating the template for existing projects who become outdated. This is where we should start: https://github.com/audreyr/cookiecutter/issues/784

**Comment by pdonorio** [comment]
We can confirm that there is no "official" updating with cookiecutter, I received [a reply on gitter](https://gitter.im/audreyr/cookiecutter?at=594afec6af2cb444279bcc0c)

**Comment by mdantonio** [comment]
Merged with #22
